LaTeX/LaTeX.sublime-syntax

```diff
@@ -552,7 +552,7 @@ contexts:
         4: variable.parameter.function.latex
         5: punctuation.definition.group.brace.end.latex
       push:
-        - meta_scope: meta.environment.math.block.be.latex
+        - meta_content_scope: meta.environment.math.block.be.latex
```

Comment:
I would not apply this to those pairs, because it would break bracket highlighter and I don't see disadvantages for it.
Since it should have the same or a special highlighting and it has only 1-2 chars and is therefore to short to use hotkeys inside. However I agree, that it might look less consistent.
Nonetheless I think this is the best solution to do it like this.